test/functional/page_objects/common_page.js

```diff
@@ -329,12 +329,6 @@ export function CommonPageProvider({ getService, getPageObjects }) {
       });
     }
 
-    async closeToast() {
```

Comment:
I think we should strive for very specific, intentional actions. A "toast" on its own is meaningless... what matters it what the toast represents, e.g. success, failure, etc. As we make changes to our UX we could replace this toast with a callout or a checkmark icon somewhere in the UI. A test which clicks a button and then looks for a toast representative of a successful outcome will be easier to understand and update than a test which just closes an open toast.